README.md

```diff
@@ -44,7 +44,7 @@ We are excited about your interest in CORTX and hope you will join us. We take o
 
 ## Get Started
 
-- Download a [CORTX virtual machine image](doc/ova/1.0.4/CORTX_on_Open_Virtual_Appliance.rst) to get up and running in minutes.
+- Download a [CORTX virtual machine image](doc/ova/2.0.0/PI-3/CORTX_on_Open_Virtual_Appliance_PI-3.rst) to get up and running in minutes.
```

Comment:
@hseow-seagate, you can update the latest OVA link as published in https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-4/CORTX_on_Open_Virtual_Appliance_PI-4.rst

Reply:
Done